src/types/registration.types.ts

```diff
@@ -79,9 +79,11 @@ interface RegistrationPublisher {
   id: string;
 }
 
-type ImportSourceName = 'Brage' | 'Cristin' | 'Scopus';
+type AdditionalIdentifierType = 'CristinIdentifier' | 'ScopusIdentifier';
```

Comment:
Det finnes også `HandleIdentifier` som brukes av Brage-import. Og `AdditionalIdentifier` er også en egen type (for bakoverkompatibilitet). Om disse må tas med er jeg usikker på.